tools/log_analyzer.py

```diff
@@ -17,6 +17,45 @@
 from randovania.games.game import RandovaniaGame
 from randovania.layout.layout_description import LayoutDescription
 
+NON_PROGRESSION = [
```

Comment:
the code previously was using item names that show up in the item order as progression things. The only things that wouldn't catch is Artifacts, but that's being inconsistent with Sky Temple Keys.

Is there any particular reason for this?

Reply:
It was needed for `regions_weighted`, because that looked at `game_modifications`. I think I can filter without this list. Let me try something.

Reply:
I was able to remove the `--progression-only` flag and this `NON_PROGRESSION` list.